assignments/callbacks.js

```diff
@@ -23,25 +23,55 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
-}
+    // getLength passes the length of the array into the callback.
+  return cb(arr.length);
+} 
+getLength(items, length => 
+  console.log(`The length of the array is ${length}.`))
```

Comment:
I love that you are using full sentences here! That will be a great habit to have.